rdtools/availability.py

```diff
@@ -531,6 +536,11 @@ def _combine_losses(self, rollup_period='M'):
             The period on which to roll up losses and calculate availability.
         """
 
+        # Allow pandas < 2.0 to use 'M' as an alias for MonthEnd
+        # https://pandas.pydata.org/docs/whatsnew/v2.2.0.html#deprecate-aliases-m-q-y-etc-in-favour-of-me-qe-ye-etc-for-offsets
+        if rollup_period == "ME":
```

Comment:
Let's update the docstrings for _combine_losses and run to explain the situation. Maybe something like "Default value "ME" triggers monthly rollup period. For other aliases, be sure they are compatible with your version of Pandas"

Reply:
Good call. I've updated the docstrings.